### DISCUSSION TITLE
Can't install manually

### DISCUSSION THREAD

**Discussion by EasyD0** [microsoft/vscode-jupyter#13593] [Questions and Answers]
I downloaded ms-toolsai.jupyter-2023.5.1001511100.vsix from the market, and installed it on linux, the result is：
Unable to install extension 'ms-toolsai.jupyter' as it is not compatible with VS Code '1.78.2'.
How to solve it? thank you.

**Comment by nevdelap** [comment]
I have the same problem. I'm always on an older version because it is tiresome to have to figure out what version is compatible. I've just installed the latest VSCode 1.82.2, and the marketplace shows these versions...

![image](https://github.com/microsoft/vscode-jupyter/assets/583807/1f71e728-7126-4cd1-9356-d43db1ca5352)

None of them are compatible. A while back I read somewhere a rationale for why  the latest version of this extension isn't compatible with the latest version of VSCode for reasons that I cannot remember, but all my other extensions don't give me this problem? What to do? 🙏

(I'm on Linux.)

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/192738
